### DISCUSSION TITLE
Comment on Steam profile

### DISCUSSION THREAD

**Discussion by HenrikhWolf** [ReactiveDrop/reactivedrop_translations#749] [General]
I got a comment on my steam profile about this translation project.
Didn't click the link, just searched it on Google and I got here.
Was that comment real?
May I help with this?

**Comment by BenLubar** [comment]
I don't know anything about the comment, but we're always happy to have more translators!

**Reply by BenLubar** [reply]
Added `latam` to the language list: https://github.com/ReactiveDrop/reactivedrop_translations/blob/master/PROGRESS.md#latam-espa%C3%B1ol-latinoamericano

**Reply by HenrikhWolf** [reply]
Oh, I'll help with it in my spare time, then :)

**Comment by Duke-M-commits** [comment]
So, what's your steam profile url? 
That one I guess https://steamcommunity.com/id/HenrikhWolf
The link is real.

**Comment by Blueberryy** [comment]
> Was that comment real?

That was pretty real! Thank you for being interested and we're happy to see if you can help with translation for your language :p

**Reply by HenrikhWolf** [reply]
haha, I was concerned because I've been receiving a lot of scam in Discord and Steam and sometimes it says something about translating a project since I was talking about translating a game in that developer's discord server.
